lib/internal/errors.js

```diff
@@ -69,6 +93,7 @@ module.exports = exports = {
   Error: makeNodeError(Error),
   TypeError: makeNodeError(TypeError),
   RangeError: makeNodeError(RangeError),
+  AssertionError: makeAssertionError(),
```

Comment:
Cool. Also there is a test case that depends on how the `AssertionError` object was created (regex matching). How do I fix that? Earlier `AssertionError` object were created like `throw new AssertionError`, but now in `assert.js`, I do it like this `throw new error.AssertionError`.